src/Emogrifier/CssInliner.php

```diff
@@ -154,7 +156,7 @@ class CssInliner extends AbstractHtmlProcessor
      *
      * @throws SyntaxErrorException
      */
-    public function inlineCss($css = '')
+    public function inlineCss(string $css = ''): self
```

Comment:
Just a query, is the coding standard specifically to have no space between `)` and `:` for the return value type hint?  (Personally, I'd prefer a space, but I can get used to its absence.)

Reply:
From the PSR-12 document:

> The colon and declaration MUST be on the same line as the argument list closing parenthesis with no spaces between the two characters.

Reply:
Fair enough :)